File: 1598-crawler-log-folder/1598-crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int cntr=0;
        for(int i=0;i<logs.size();i++){
            if(logs[i][0]=='.' && logs[i][1]=='/')continue;
            else if(logs[i][0]=='.' && logs[i][1]=='.'){
                cntr=max(0, cntr-1);
            }
            else cntr++;
        }
        return cntr;
    }
};

File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        cntr=0
        for x in logs:
            if x[0]=='.' and x[1]=='.':
                if cntr>0:
                    cntr=cntr-1
            elif x[0]=='.' and x[1]=='/':
                continue;
            else:
                cntr=cntr+1
                
        return cntr